modules/olm-creating-fb-catalog-image.adoc

```diff
@@ -113,15 +111,15 @@ name: preview
 entries:
   - name: <operator_name>.v0.1.0 <1>
 ----
-<1> Ensure that you include the period (`.`) after `<operator_name>` but before the `v` in the version. Otherwise, the entry will fail to pass the `opm validate` command.
+<1> Ensure that you include the period (`.`) after `<operator_name>` but before the `v` in the version. Otherwise, the entry fails to pass the `opm validate` command.
```

Comment:
In the context of this PR, this change is fine.  However, I feel like this is a best-practices discussion about package naming and feels somewhat out of place in this section.
@joelanford do you know if there was a specific reason to add this callout here (in the original doc)?

Reply:
This doesn't block this PR.  I was just trying to get a little background to determine if we should create a new issue to remove this